app/view/window/text/FacsimileView.js

```diff
@@ -23,7 +23,6 @@ Ext.define("EdiromOnline.view.window.text.FacsimileView", {
     requires: [
         "EdiromOnline.view.window.image.ImageViewer",
         "EdiromOnline.view.window.image.OpenSeaDragonViewer",
```

Comment:
Comma should be removed:
```suggestion
        "EdiromOnline.view.window.image.OpenSeaDragonViewer"
```